src/core/core-varlink.c

```diff
@@ -134,12 +134,12 @@ static int build_group_json(const char *group_name, gid_t gid, JsonVariant **ret
         assert(ret);
 
         return json_build(ret, JSON_BUILD_OBJECT(
-                                          JSON_BUILD_PAIR("record", JSON_BUILD_OBJECT(
-                                                                          JSON_BUILD_PAIR("groupName", JSON_BUILD_STRING(group_name)),
-                                                                          JSON_BUILD_PAIR("gid", JSON_BUILD_UNSIGNED(gid)),
-                                                                          JSON_BUILD_PAIR("service", JSON_BUILD_STRING("io.systemd.DynamicUser")),
-                                                                          JSON_BUILD_PAIR("disposition", JSON_BUILD_STRING("dynamic"))))));
-}
+                                   JSON_BUILD_PAIR("record", JSON_BUILD_OBJECT(
+                                       JSON_BUILD_PAIR("groupName", JSON_BUILD_STRING(group_name)),
+                                       JSON_BUILD_PAIR("gid", JSON_BUILD_UNSIGNED(gid)),
+                                       JSON_BUILD_PAIR("service", JSON_BUILD_STRING("io.systemd.DynamicUser")),
+                                       JSON_BUILD_PAIR("disposition", JSON_BUILD_STRING("dynamic"))))));
+    }
```

Comment:
but but but, emacs wanted it this way? are you doubting emacs? ;-)

Reply:
It's just not practical.... I wish we had a way to tell emacs to indent this case less rigidly.